src/wizard/Stepper.tsx

```diff
@@ -27,7 +27,7 @@ export const Stepper: React.FC<StepperProps> = ({
             {steps.map((step, index) => (
                 <Step
                     key={step.key}
-                    completed={false}
+                    completed={step.completed || false}
```

Comment:
It's 100% ok, just as a note, other options for boolean defaults: `Boolean(step.completed)`, `step.completed ?? false`.